helm/amazon-ec2-metadata-mock/templates/daemonset.yaml

```diff
@@ -43,6 +43,10 @@ spec:
                 - amd64
                 - arm
                 - arm64
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
```

Comment:
I believe this line will give you more than 8 indentations since its already indented. 

When using `nindent` I think you would want to make it look like L#29 to get exactly 8 indentations.

Reply:
+1 ; will update tolerations block to reflect this